tests/test_helpers/test_dialogue/test_base.py

```diff
@@ -81,17 +82,9 @@ def __init__(
             agent_address=agent_address,
             role=role,
             message_class=message_class,
-            rules=BaseDialogue.Rules(
-                cast(FrozenSet[Message.Performative], self.INITIAL_PERFORMATIVES),
-                cast(FrozenSet[Message.Performative], self.TERMINAL_PERFORMATIVES),
-                cast(
-                    Dict[Message.Performative, FrozenSet[Message.Performative]],
-                    self.VALID_REPLIES,
-                ),
-            ),
         )
 
-    def is_valid(self, message: Message) -> bool:
+    def is_valid(self, message: Message) -> Tuple[bool, str]:
```

Comment:
Can we actually rename this (as discussed above) and make it a default implementation in base class. Then if the user wants to; they can always override it? Saves us having to have this unnecessary code here.

Reply:
I'll rename it to `custom_validation`

Reply:
Done